wordpress-mobile/WordPressAuthenticator-iOS
# Creating 'Unified Auth' folder.

Ref #182 

This creates a `Unified Auth` folder in the workspace to house shiny new Unified Auth files. The inaugural entry is `GoogleAuthenticator`.

<img width="237" alt="Screen Shot 2020-05-27 at 5 24 41 PM" src="https://user-images.githubusercontent.com/1816888/83081935-21e10400-a03f-11ea-95f4-c6bd741f5b98.png">

To test:
- Point your local WP to this branch.
- Run the app.
- Verify Google login works. Rejoice.
